include/DataFile.h

```diff
@@ -113,9 +113,8 @@ class LMMS_EXPORT DataFile : public QDomDocument
 	void upgrade_1_1_0();
 	void upgrade_1_1_91();
 	void upgrade_1_2_0_rc3();
-	void upgrade_1_2_0_rc2_42();
-	void upgrade_1_3_0_alpha_1();
 	void upgrade_1_3_0();
+	void upgrade_noHiddenClipNames();
```

Comment:
The removal of `rc2_42` isn't part of this PR, it's from the merge with master. `alpha_1` (added in this PR) was renamed to `noHiddenClipNames` and moved to the bottom for consistency. Since it's the newest upgrade routine it needs to be last in the upgrade routine list, so it makes sense for it to be last here as well.